src/library_webgpu.js

```diff
@@ -1532,21 +1536,29 @@ var LibraryWebGPU = {
     function makeDepthStencilAttachment(dsaPtr) {
       if (dsaPtr === 0) return undefined;
 
+      var depthLoadOpInt = WebGPU.LoadOp[
+        {{{ gpu.makeGetU32('dsaPtr', C_STRUCTS.WGPURenderPassDepthStencilAttachment.depthLoadOp) }}}];
+      var depthClearValue = {{{ makeGetValue('dsaPtr', C_STRUCTS.WGPURenderPassDepthStencilAttachment.depthClearValue, 'float') }}};
+      var stencilLoadOpInt = WebGPU.LoadOp[
```

Comment:
I think you want to remove the `WebGPU.LoadOp[ ]` here (in keeping with the "Int" variable name) and move it below. Then the depthLoadValue/stencilLoadValue rows can compare with the integer enum value instead of the string.